app/views/citizens/check_answers/index.html.erb

```diff
@@ -51,6 +51,8 @@
           ),
           read_only: false) %>
 
+  <%= render 'student_finance' %>
```

Comment:
We only want to do this if the feature flag on the settings table is set to use new student loan - otherwise we'll get a no to the answer for "do you get student finance?" even though it's specified as part of other income.